code-studio/src/js/components/IconLibrary.jsx

```diff
@@ -47,7 +47,7 @@ var IconLibrary = React.createClass({
           <input
             onChange={this.search}
             style={styles.input}
-            placeholder={msg.iconSearchPlaceholder()}/>
+            placeholder={window.dashboard.i18n.t('components.icon_library.search_placeholder')}/>
```

Comment:
Maybe better to do `dashboard.i18n.t` and add a `/* global dashboard */` at the top of the file to track this dependency.